Source/SwiftLintFramework/Extensions/SyntaxKind+SwiftLint.swift

```diff
@@ -9,6 +9,15 @@
 import SourceKittenFramework
 
 extension SyntaxKind {
+    // swiftlint:disable:next function_body_length
```

Comment:
Good catch. I'll delete that. It was a holdover from a more complicated implementation of this.